Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -443,8 +443,7 @@ ms::TransformPtr msblenContext::exportReference(msblenContextState& state, msble
 
             auto do_merge = [this, dst, &dst_mesh, &src_mesh, &settings, &state]() {
                 dst_mesh.merge(src_mesh);
-                if (settings.ExportSceneCache)
-                    dst_mesh.detach();
```

Comment:
@sindharta do you remember why you introduced this if statement? I was wondering if you had to deal with the same problem in the past when exporting references in the scene cache.

Reply:
Let’s discuss on Slack.